apps/e2e/lists-storybook-e2e/src/e2e/infinite-scroll.component.cy.ts

```diff
@@ -0,0 +1,34 @@
+import { E2eVariations } from '@skyux-sdk/e2e-schematics';
+
+describe('lists-storybook', () => {
```

Comment:
need to add the "infinite scroll" to the describe

Reply:
Fixed